Cargo.toml

```diff
@@ -29,6 +29,7 @@ members = [
     "utils/process",
     "utils/transfer",
     "utils/path",
+    "utils/constraints",
```

Comment:
Note that there's utils/agreement-utils crate. It would be nice to have all utilities for offers/demand building in one place. But it's not a must.

Reply:
Good idea.

Reply:
Done!